src/Server/SSHPipeInterface.php

```diff
@@ -0,0 +1,19 @@
+<?php
+/* (c) Anton Medvedev <anton@medv.io>, Maxim Kuznetsov <skypluseg@gmail.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Deployer\Server;
+
+interface SSHPipeInterface
```

Comment:
@elfet I don't understand what I should do here. My position is written down above. I think that name of the interface might be better but I have no idea.